src/mmw/js/src/analyze/views.js

```diff
@@ -222,16 +228,25 @@ var TabContentsView = Marionette.CollectionView.extend({
 var TableRowView = Marionette.ItemView.extend({
     tagName: 'tr',
     template: tableRowTmpl,
-        templateHelpers: function() {
+    templateHelpers: function() {
+        var quotient = this.options.kiloMeters ? M_IN_KM : 1;
+
         return {
             // Convert coverage to percentage for display.
-            coveragePct: (this.model.get('coverage') * 100).toFixed(1)
+            coveragePct: (this.model.get('coverage') * 100).toFixed(1),
```

Comment:
We handle display formatting for scaledArea in the template using nunjucks filters. It may make sense to do the same thing for coveragePct so that the logic is in the same place.